### ISSUE TITLE
Hello! Is about the style.md file on the main repository 

### ISSUE THREAD

**Issue by MetalMaxMX** [bakpakin/Fennel#478]
Hello!

We are working on adding Fennel to the FSF directory-listing, we ran a scan and analysis of the source code and we found that it has indeed met the licensing requirements in order to be added except for one file, that it being a markdown file by the name of style.md, which is licensed under the CC BY-NC-SA 3.0. 

This file alone keeps us from adding it to the FSF directory-listing, as a result, we would like to ask how this file came to be and if something can be done to have a potential replacement or anything similar so that we can add and file this project as approved. 

Cheers!
MetalMaxMX.

**Comment by technomancy** [comment]
We mentioned this on IRC, but the style guide is a derivative work of Taylor Campbell's Lisp Style Guide: http://mumble.net/~campbell/scheme/style.txt

As such we cannot relicense it on our own.

However, unlike most of the documentation, this file is not included in the installation process of Fennel, so it shouldn't affect users of Fennel. I don't know the specific requirements of the directory, but unless its criteria must apply to every single file in the program's repository, it shouldn't cause any problems regarding the freedom of users.

**Comment by MetalMaxMX** [comment]
Hello!

Yes, the criteria of the FSF directory-listing applies to every file in a program's repository. More specifically, we work with tagged releases. We primarily concern ourselves that a tagged release is free as in freedom in order to properly tell others which package is fully free software, the reason why we don't point specifically to a main branch has to do with files constantly changing and potentially even licenses doing so, so we evaluate in terms of those tagged releases as stated.

I emailed the author of that style.txt asking them if they were willing to re-license that style guide, as that seemed to be the only point of contact left by them. If we ever hear of them and they give us permission to relicense that specific file, we'll let you know. 

We also discussed the question of what to do with this file on IRC as things stood (#fsf at Libera.Chat), we were wondering if it was possible that the repository could strip the file but tell users where to find it elsewhere as we initially thought that it was unmodified, however, as it contains many modifications to it in order to suit Fennel's needs, we realized this couldn't be done either. The conclusion we came to was that I would temporarily fork the repository, remove style.md and mentions of it and pack it as a tagged release in order for it to work as intended on the FSF directory-listing entry, and that if we do hear from the author of style.txt, and that they were willing to relicense, then that we'd let you know so that style.md can also be modified to bring in the relicensing and then point to a tagged release of upstream as a whole. 

That's where things stand for now. If you have any ideas or the like in order to solve this issue or have an alternate take, we'd like to know! 

Cheers!
MetalMaxMX.

**Comment by technomancy** [comment]
No, I don't think there's anything we can do until we hear back from Riastradh.

However, I've updated the readme to make the licensing clearer at a glance.

**Comment by technomancy** [comment]
It doesn't look like this is going to happen, so I think it's best to close this out since we can't do anything about it.
